po/es.po

```diff
@@ -399,7 +406,7 @@ msgstr "Obsoleto"
 
 #: pacaur:1304
 msgid "Submitted"
-msgstr "Enviado"
+msgstr "Encargado"
```

Comment:
This translation would be 'Primer encargado' in Spanish based on AUR webpage not 'Encargado' (Maintainer).